### PR TITLE
feat: added sorting for events

### PR DESCRIPTION
**Pull request by BugGambit**
Our API now supports sorting on events:
https://docs.cognite.com/api/v1/#operation/listEvents
https://docs.cognite.com/api/v1/#operation/advancedListEvents

### CODE REVIEW

**Comment by polomani** [comment]
wow  you can Python??

File: cognite/client/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 from cognite.client._cognite_client import CogniteClient
 
-__version__ = "1.3.1"
+__version__ = "1.4.0"
```
[the remark this answers is not in the document]
Comment (sanderland) [reply cognite/client/__init__.py]: and add this feature to changelog anyway :D

**Comment by sanderland** [comment]
and your jenkins error is related to our ever uglier growing list of hardcoded exceptions in /tests/tests_unit/test_api/test_signatures.py

File: cognite/client/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (erlendvollset) [reply cognite/client/__init__.py]: Should not have breaking changes in minor. We have made some exceptions to that, but only after ensuring that whoever was using the feature knew about it.

[the remark this answers is not in the document]
Comment (polomani) [reply cognite/client/__init__.py]: that's we have an automated process around this in js sdk)))

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,8 +21,11 @@ Changes are grouped as follows
...
 ## [Unreleased]
 
+## [1.3.2] - 2019-10-16
+
 ### Added
 - Datapoints objects now store is_string, is_step and unit to allow for better interpretation of the data.
+- Sorting when listing events
 - Added a search function in the relationships API.
 
 ### Changed
```
Comment (sanderland) [inline CHANGELOG.md]: please merge changed sections

File: cognite/client/_api_client.py [hunk 3]
```diff
@@ -299,9 +300,13 @@ def _list_generator(
                 params = filter.copy()
                 params["limit"] = current_limit
                 params["cursor"] = next_cursor
+                if sort is not None:
```
Comment (sanderland) [inline cognite/client/_api_client.py]: just add to body without checking is fine

**Pull request merged**